### PR TITLE
v158: Enabling pagination of `account_history` page

### LINKED ISSUE #5042: v158: account_history page no longer has pagination
In previous versions of Zen Cart, the `account_history` page limits the count of orders displayed to 'MAX_DISPLAY_ORDER_HISTORY' and paginates.

In zc158, the page now displays all orders with no pagination available since the history information is returned by a call to `Customer::getOrderHistory`.  That class method doesn't export the SQL used for the query, so it's not possible to pass that on to the split_page_results class.

I'll note also that the Bootstrap template counts on that base SQL to be returned so that it can use its own split_page_results class to format the output using Bootstrap classes.

### PR DESCRIPTION
**Pull request by lat9**
Fixes #5042

### CODE REVIEW

**Comment by scottcwilson** [comment]
This is a mandatory change for all templates, right?  Could we avoid this by creating a new method in the Customer class that would return data in the 157 format?  Does doing that make sense?

**Comment by lat9** [comment]
That's actually not a _change_ to the templates, it's correcting the zc158 version to perform the pagination that has occurred in all (?) previous `template_default` versions.

**Comment by scottcwilson** [comment]
Sorry, let me rephrase: The query in `includes/modules/pages/account_history/header_php.php` has changed.  Will an old template still work with the new data returned by this query?

**Comment by lat9** [comment]
They will _now_.  There was a change required to the Bootstrap template's additional `header_php_`, but the template itself did not require change
